### PR TITLE
About page michael

### PR DESCRIPTION
**Pull request by kaka5277**
imageimage

### CODE REVIEW

File: about.html
Comment (davidtaing) [inline about.html]: This link isn't working. Try out `mailto:<email-address>`.

Source: https://css-tricks.com/snippets/html/mailto-links/

File: about.html
Comment (davidtaing) [inline about.html]: This is a brand new company, so we won't be able to claim that 1000 companies trust us. I feel that it's better to remove this line for now.

Also look into the career form section above (lines 255 to 313) and look at modifying/removing that section. _I'm not able to highlight them, but I've left a screenshot below._

image

File: about.html
Comment (davidtaing) [inline about.html]: Can you please format this as follows?

image

Source: https://www.stylemanual.gov.au/grammar-punctuation-and-conventions/numbers-and-measurements/telephone-numbers#use_chunks_of_digits_to_help_people_read_telephone_numbers

File: about.html
Comment (davidtaing) [inline about.html]: Looks like we need to make this clickable. Can you please wrap this email address inside an `<a>` tag with a `href="mailto:<email-address>"` property?

**Comment by davidtaing** [comment]
Hold off on making changes for a sec. I'll fix up the merge conflict for you here first and will make some changes to this branch.

**Review by davidtaing — CHANGES_REQUESTED** [review]
Thanks for your PR @kaka5277.

- I've left some comments above, please review these when you get a chance.
- **[Important]** I've also merged `main` into this branch and fixed up the merge conflict. This means that I've made changes to your branch, and you will have to sync up with my changes.

  In order synchronize my changes with yours, please run `$ git pull` in your terminal.


- Finally, I noticed that the branch wasn't synchronized with main before new changes were added. This can cause problems such as the merge conflict from before. Fortunately, there haven't been too many changes, so the merge conflict was not bad.

  Linking to my comment to @ZizhangOu since its relevant: 
https://github.com/codesydney/aicta/pull/9#issuecomment-1474730658

  I'll try to create some diagrams to make this easier to understand. Those will be added in future comments

**Comment by davidtaing** [comment]
When we merge two branches, git will create a merge commit. So after `about-page-michael` was merged into `main`, there was a new merge commit added to main.

Let's say we made two commits `B` & `C` to the `about-page`. The merge added a merge commit of `D`. So we can say the history of the `about-page` file in `main` is:

`B` -> `C` -> `D`.

When additional changes were added to the `about-page` file in the `about-page-michael` branch, we did not have `D` in our history. So in this branch we had a history of:

`B` -> `C` -> `E`.

This is why there was a merge conflict.

Fortunately, not much has changed, so it was not bad. But imagine if you had 300+ lines of code on one branch and another 300+ lines of code in the other. That can be extremely difficult to fix.

### Without creating a new branch/synchronizing with main.
untitled(20)

---

Creating a new branch is almost always the best way to go. If you would like to reuse the branch's name, I'll suggest deleting the branch after you merge and creating a new one with the same name.

Not only do you capture your merge commit, but it also capture any changes that other developers have made to main.

### With new branch or synchronizing with main.
untitled(21)

If you want to go in depth, here's a good 1 hour 40 mins talk on Git:
https://www.youtube.com/watch?v=1ffBJ4sVUb4

---

Pinging you guys to loop you in. @kaka5277 @ZizhangOu 

Feel free to ask any follow-up questions. 

Even for those not directly involved in this thread, you're more then welcome to ask. e.g. Code.Sydney member, or general GitHub user. :wink:

**Comment by kaka5277** [comment]
Hi Dave,
&nbsp;Thank you so much for not only reviewing for me ,but also providing so detailed documents about Git skills.really appreciate your time and help.I'll have a look and learn it.
Michael




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "codesydney/aicta"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月1日(星期六) 中午11:35
***@***.***&gt;;
抄送:&nbsp;"Jun ***@***.******@***.***&gt;;
主题:&nbsp;Re: [codesydney/aicta] About page michael (PR #11)





 
When we merge two branches, git will create a merge commit. So after about-page-michael was merged into main, there was a new merge commit added to main.
 
Let's say we made two commits B &amp; C to the about-page. The merge added a merge commit of D. So we can say the history of the about-page file in main is:
 
B -&gt; C -&gt; D.
 
When additional changes were added to the about-page file in the about-page-michael branch, we did not have D in our history. So in this branch we had a history of:
 
B -&gt; C -&gt; E.
 
This is why there was a merge conflict.
 
Fortunately, not much has changed, so it was not bad. But imagine if you had 300+ lines of code on one branch and another 300+ lines of code in the other. That can be extremely difficult to fix.
 
Without creating a new branch/synchronizing with main.
 

 
 
Creating a new branch is almost always the best way to go. If you would like to reuse the branch's name, I'll suggest deleting the branch after you merge and creating a new one with the same name.
 
Not only do you capture your merge commit, but it also capture any changes that other developers have made to main.
 
With new branch or synchronizing with main.
 

 
If you want to go in depth, here's a good 1 hour 40 mins talk on Git:
 https://www.youtube.com/watch?v=1ffBJ4sVUb4
 
 
Pinging you guys to loop you in. @kaka5277 @ZizhangOu
 
Feel free to ask any follow-up questions.
 
Even for those not directly involved in this thread, you're more then welcome to ask. e.g. Code.Sydney member, or general GitHub user. 😉

File: about.html
Comment (davidtaing) [reply about.html]: Thanks for doing that Michael. Just need to go to Line 321 as well. Can you remove the subheadline here since it says 1000 companies trust AICTA?

**Comment by davidtaing** [comment]
@kaka5277, please let me know when this is ready for another review. Thanks

**Comment by kaka5277** [comment]
hi dave,it is already for review.

**Review by davidtaing — APPROVED** [review]
Looks good, great work.

Love that you broke up the changes into multiple commits.

Approved @kaka5277.

**Comment by kaka5277** [comment]
thank you so much.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "codesydney/aicta"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年4月3日(星期一) 晚上8:55
***@***.***&gt;;
抄送:&nbsp;"Jun ***@***.******@***.***&gt;;
主题:&nbsp;Re: [codesydney/aicta] About page michael (PR #11)





 
Merged #11 into main.

**Pull request merged**
